lib/matplotlib/animation.py

```diff
@@ -1613,10 +1613,14 @@ def init_func() -> iterable_of_artists
        blitting any animated artists will be drawn according to their zorder.
        However, they will be drawn on top of any previous artists, regardless
        of their zorder.  Defaults to *False*.
+
+    cache_frame_data : bool, optional
+       Controls whether frame data is cached. Defaults to *True*.
+       Disabling cache might be helpful when frames contain large objects.
     """
 
     def __init__(self, fig, func, frames=None, init_func=None, fargs=None,
-                 save_count=None, **kwargs):
+                 save_count=None, cache_frame_data=True, **kwargs):
```

Comment:
could be made keyword-only.

Reply:
@anntzer Thanks for the hint.
I've applied your suggestion in the rebased commit.